examples/websocket-retry/websocket_service.out

```diff
@@ -0,0 +1,15 @@
+# To start the service, navigate to the directory that contains the
+# `.bal` file, and use the `ballerina run` command.
+$ ballerina run webSocket_service.bal
+
+[ballerina/http] started HTTP/WS listener 0.0.0.0:9095
+2019-11-20 13:31:38,842 INFO  [] - WebSocket client connected wih the server. The Connection ID: 08d40cfffe258b13-00000bfd-00000001-a1eade696304c59e-7f64181e
+
+# To stop the running service, press Ctrl + C.
+[ballerina/http] stopped HTTP/WS listener 0.0.0.0:9095
+
+# Restart the service.
+$ ballerina run webSocket_service.bal
```

Comment:
L3 and here mention web**S**ocket_service.bal but the file is web**s**ocket_service.bal